devel/capstone/Portfile

```diff
@@ -4,6 +4,7 @@ PortSystem          1.0
 PortGroup           github 1.0
 
 github.setup        aquynh capstone 3.0.5
+revision            1
 categories          devel
 platforms           darwin
 maintainers         gmail.com:aquynh
```

Comment:
You don't know his github handle, do you?

Reply:
Looks like it's @aquynh; I updated the Portfile with that.

Reply:
Thanks!